plugins/@grouparoo/cloudwatch/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "author": "Grouparoo Inc <hello@grouparoo.com>",
+  "name": "@grouparoo/cloudwatch",
+  "description": "The Grouparoo Cloudwatch Plugin",
+  "version": "0.7.0-alpha.7",
+  "license": "MPL-2.0",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "engines": {
+    "node": ">=12.0.0"
+  },
+  "homepage": "https://www.grouparoo.com",
+  "bugs": {
+    "url": "https://github.com/grouparoo/grouparoo/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/grouparoo/grouparoo.git",
+    "directory": "plugins/@grouparoo/cloudwatch"
+  },
+  "scripts": {
+    "prepare": "rm -rf dist && tsc --declaration",
+    "test": "echo 'no tests'",
```

Comment:
I'm not really shore how to test this.  I also don't know if it is required, since we are just relying on the library.